draft-packages/popover/KaizenDraft/Popover/styles.scss

```diff
@@ -130,26 +146,26 @@ $large-width: 450px;
 }
 
 .informativeIcon {
-  color: $kz-color-cluny-500;
+  color: $kz-var-color-cluny-500;
 }
 
 .positiveIcon {
-  color: $kz-color-seedling-400;
+  color: $kz-var-color-seedling-400;
 }
 
 .negativeIcon {
-  color: $kz-color-coral-500;
+  color: $kz-var-color-coral-500;
 }
 
 .cautionaryIcon {
-  color: $kz-color-yuzu-500;
+  color: $kz-var-color-yuzu-500;
 }
 
 .close {
   @include button-reset;
   margin-left: auto;
   display: inherit;
-  color: rgba(black, 0.5);
+  color: rgba($kz-var-color-wisteria-800, 0.7);
```

Comment:
This will need to be an `rgb-params` variable.

Also do we want a fallback? (black in Zen, Wisteria 800 in Heart)

Reply:
Thanks for picking this up, so easy to miss, heck.
Black was actually originally an incorrect color (going off the Zen UI Kit in figma) and should have been wisteria 800 at 75% opacity, and at the sacrifice of a 5% accuracy in opacity i'm just using 70% for both Zen and Heart 🙃

Reply:
Beautiful, thanks